test/TestCase.cpp

```diff
@@ -139,3 +138,37 @@ EVMVersionRestrictedTestCase::EVMVersionRestrictedTestCase(std::string const& _f
 	if (!comparisonResult)
 		m_shouldRun = false;
 }
+
+void EVMVersionRestrictedTestCase::processCompileToFlag()
+{
+	std::optional<uint8_t> eofVersion = solidity::test::CommonOptions::get().eofVersion();
+
+	std::string compileToString = m_reader.stringSetting("bytecodeFormat", "any");
```

Comment:
Let's make it default to `legacy`, like you suggested on the chat.

```suggestion
	std::string compileToString = m_reader.stringSetting("bytecodeFormat", "legacy");
```